src/components/ContextMenu.scss

```diff
@@ -73,4 +73,15 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media screen and (max-width: 1000px) {
```

Comment:
please change to:
``@media #{$media-query} { ... } ``  instead of defining an own query.

then the changing-width is all the same in the app.

(just search for ``@media #{$media-query}``and you will see it's used in many components)